D festeriet (#595)
datasektionen/bawang-content: namnder/eventorganet/d-festeriet/body_en.md

# D-festeriet

> D-Fest's tasks include - Organizing two parties for other officials per year, one after the Glögg-SM and one after the Val-SM. The purpose of the parties is to honor the chapter's outgoing and incoming officials. - Organize regular team building for the chapter's officials. - Possibly organize other meetings and events together with other chapters and student unions to represent the chapter externally.

Reviewer: This looks like a list that has lost it formatting when copying

```suggestion
D-Festeriet's tasks include: 

- Organizing two parties for other officials per year, one after the Glögg-SM and one after the Val-SM. The purpose of the parties is to honor the chapter's outgoing and incoming officials. 

- Organize regular team building for the chapter's officials. 

- Possibly organize other meetings and events together with other chapters and student unions to represent the chapter externally.
```

Also in the swedish version there is some stuff about haltidsfesten that's missing in the English translation

# D-festeriet

D-Festeriet's tasks include: 

> - Organizing two parties for other officials per year, one after the Glögg-SM and one after the Val-SM. The purpose of the parties is to honor the chapter's outgoing and incoming officials. 

Reviewer 2: ```suggestion
- Organizing two parties for other officials per year, one after the Vinter-VM and one after the Sommar-VM. The purpose of the parties is to honor the chapter's outgoing and incoming officials. 
```

- Organize regular team building for the chapter's officials. 

- Possibly organize other meetings and events together with other chapters and student unions to represent the chapter externally.

> Okay, maximum brainrot now. D-festeriet? Glitched. Skibidi Ohio level Bombardino. Not KTH, just pure brainrot. Got it? No? Based.
> 
> Crew: Gigetto the Little Dragon eats money. "MAMMA MIA, DELICIOUS!" Sigma mewing 24/7. Bombardino Crocodillo? The king of rizz. Dances non-stop. Screams "PIZZA PASTA MANDOLINO ATTENZIONE GRIMACE SHAKE!" Peperina Pasticcina inspects your gyatt. Finds nothing? Cooked. She screams "CHE CASINO!" Riccio the Hedgehog? Just vine boom and phonk. ENOUGH!
> 
> Meetings? Silent looksmaxxing. Who's got the sharpest jawline? "Skibidi?" "Dop Dop Yes!" Minutes? Capybara emoji. Fantastic.
> 
> Parties: Ohio final boss. LET'S GO! Walks in. BAM! Bombardino dance. Music? Skibidi Italian Phonk. NPCs glitch out completely. Zero rizz. Just falling over. "GYATT!" Fanum Tax on your soul. Chairperson? Capybara says "Ok I pull up." Peperina judges everyone. You’re cooked. Everyone screams "CHE CASINO!" Pure, unfiltered, maximum D-festeriet brainrot. Skibidi. Mamma Mia. Dop dop. Done.

Reviewer 2: ```suggestion
Okay, maximum brainrot now. D-festeriet? Glitched. Skibidi Ohio level Bombardino. Not KTH, just pure brainrot. Got it? No? Based.
```
